Dockerfile.template

```diff
@@ -27,6 +27,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && case "${dpkgArch##*-}" in \
     amd64) ARCH='x64';; \
     ppc64el) ARCH='ppc64le';; \
+    arm64v8) ARCH='arm64';; \
```

Comment:
Here's some output from a real `arm64` box running Docker:
```console
root@rpi3:~# uname -m
aarch64
root@rpi3:~# docker run --rm arm64v8/debian dpkg --print-architecture
arm64
root@rpi3:~# docker run --rm arm64v8/alpine apk --print-arch
aarch64
```

(and `library/node` over in https://github.com/docker-library/official-images/blob/5ff8e195b3db1a97e58497e3c5077be09ae2b7de/library/node needs `arm64v8`)